File: String/3110. Score of a String/ScoreOfString.java
public class ScoreOfString {
    public int scoreOfString(String s) {
        int score=0;
        for(int i=0;i<s.length()-1;i++){
            if (s.charAt(i) > s.charAt(i + 1)) {
                score += s.charAt(i) - s.charAt(i + 1);
            } else {
                score += s.charAt(i + 1) - s.charAt(i);
            }
        }
        return score;
    }
    public static void main(String[] args){
        ScoreOfString sol = new ScoreOfString();
        System.out.println(sol.scoreOfString("hello"));
        System.out.println(sol.scoreOfString("zaz"));
    }
}

File: String/3110. Score of a String/ScoreOfString.go
package main

import (
	"fmt"
)

func scoreOfString(s string) int {
	score := 0
	for i := 0; i < len(s)-1; i++ {
		if s[i] > s[i+1] {
			score += int(s[i]) - int(s[i+1])
		} else {
			score += int(s[i+1]) - int(s[i])
		}
	}
	return score
}

func main() {
	fmt.Println(scoreOfString("hello")) // 输出结果
	fmt.Println(scoreOfString("zaz"))   // 输出结果
}
